### PR TITLE
[JENKINS-63274] Fix regression on copyButton due to recent CSS changes

### PR DESCRIPTION
**Pull request by Wadeck**
See [JENKINS-63274](https://issues.jenkins-ci.org/browse/JENKINS-63274). After #4700 (2.238+) the copy button was no longer working as the temporary textArea created for the copy was invisible (height=0) instead of being of size 1x1.

You can find one example of the copy button in the configuration page of the user, after having generated an ApiToken.

This behavior was introduced in 2.173+ and broken since 2.238+.

⚠️ I was not able to find anything supported in HtmlUnit to unit test the behavior to prevent a new regression in the future.

### Proposed changelog entries

* JENKINS-63274, API token copy to clipboard button works again (regression in 2.238)

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [x] (If applicable) Jira issue is well described
- [x] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [x] ❌Appropriate autotests or ✔️ explanation to why this change has no tests
- [n/a] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

**Review by timja — APPROVED** [review]

**Comment by batmat** [comment]
Would this be ATH-able, if not doable through HtmlUnit as you are saying?

**Comment by Wadeck** [comment]
> Would this be ATH-able

I expect Selenium to support such use case. Up to Felix when he's back from holidays, I was just unblocking the situation there.

File: core/src/main/resources/lib/layout/copyButton/copyButton.js
Comment (fqueiruga) [inline core/src/main/resources/lib/layout/copyButton/copyButton.js:17]: tabindex is not style, is an attribute and should be set with `setAttribute` as a string https://developer.mozilla.org/en-US/docs/Web/API/Element/setAttribute

**Review by fqueiruga — CHANGES_REQUESTED** [review]

File: core/src/main/resources/lib/layout/copyButton/copyButton.js
Comment (Wadeck) [inline core/src/main/resources/lib/layout/copyButton/copyButton.js:17]:
```suggestion
        el.setAttribute("tabindex", "-1");
```

**Review by oleg-nenashev — APPROVED** [review]
LGTM, as long as it is tested. I wonder why it was not reported by users, maybe we need to promote the new feature a bit more

**Comment by timja** [comment]
> LGTM, as long as it is tested. I wonder why it was not reported by users, maybe we need to promote the new feature a bit more

I tried using it 😂 
The first couple of times you think it's user error or maybe something wrong with your browser.

**Comment by daniel-beck** [comment]
> I wonder why it was not reported by users, maybe we need to promote the new feature a bit more

GitHub has a similar feature and TBH I just select the token and copy it to clipboard manually 🤷 I'm not surprised.

**Comment by timja** [comment]
I've verified it locally via core-pr tester, this fixes it...

**Review by NicuPascu — APPROVED** [review]
LGTM

**Comment by oleg-nenashev** [comment]
We may merge it in 24 hours if there is no negative feedback. Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process

**Comment by Wadeck** [comment]
For increased confidentiality, we could mask the default token, and letting people doing the copy-paste manually or reveal the content, it could be useful depending on the context, but too much for this PR ;)

**Comment by timja** [comment]
That’s what azure key vault does ^.

**Pull request merged**
